clang/test/dpct/cusparse-after101.cu

```diff
@@ -0,0 +1,24 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0
+// RUN: dpct --format-range=none --out-root %T/cusparse-after101 %s --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck --input-file %T/cusparse-after101/cusparse-after101.dp.cpp --match-full-lines %s
+
+#include <cusparse_v2.h>
+
+void foo1() {
+  cusparseHandle_t handle;
+  float* a_val;
+  int* a_row_ptr;
+  int* a_col_ind;
+  float* b_val;
+  int* b_col_ptr;
+  int* b_row_ind;
+
+  size_t ws_size = 0;
+  void *ws;
+
+  // CHECK: ws_size = 0;
+  // CHECK-NEXT: dpct::sparse::csr2csc(*handle, 3, 4, a_val, a_row_ptr, a_col_ind, b_val, b_col_ptr, b_row_ind, dpct::library_data_t::real_float, oneapi::mkl::index_base::zero);
+  cusparseCsr2cscEx2_bufferSize(handle, 3, 4, 7, a_val, a_row_ptr, a_col_ind, b_val, b_col_ptr, b_row_ind, CUDA_R_32F, CUSPARSE_ACTION_NUMERIC, CUSPARSE_INDEX_BASE_ZERO, CUSPARSE_CSR2CSC_ALG1, &ws_size);
+  cusparseCsr2cscEx2(handle, 3, 4, 7, a_val, a_row_ptr, a_col_ind, b_val, b_col_ptr, b_row_ind, CUDA_R_32F, CUSPARSE_ACTION_NUMERIC, CUSPARSE_INDEX_BASE_ZERO, CUSPARSE_CSR2CSC_ALG1, ws);
```

Comment:
what is the migration if CUSPARSE_ACTION_NUMERIC ==> CUSPARSE_ACTION_SYMBOLIC?

Reply:
done